executor/distsql_test.go

```diff
@@ -21,23 +21,25 @@ import (
 	"math/rand"
 	"runtime/pprof"
 	"strings"
+	"testing"
 	"time"
 
-	. "github.com/pingcap/check"
 	"github.com/pingcap/tidb/domain"
 	"github.com/pingcap/tidb/executor"
 	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/parser/model"
 	"github.com/pingcap/tidb/store/copr"
+	"github.com/pingcap/tidb/store/mockstore"
 	"github.com/pingcap/tidb/table/tables"
 	"github.com/pingcap/tidb/tablecodec"
+	"github.com/pingcap/tidb/testkit"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/israce"
 	"github.com/pingcap/tidb/util/mock"
-	"github.com/pingcap/tidb/util/testkit"
+	"github.com/stretchr/testify/require"
+	"github.com/tikv/client-go/v2/testutils"
 )
 
-// nolint:unused
 func checkGoroutineExists(keyword string) bool {
```

Comment:
it can remove because no one uses it.

Reply:
you mean removing the checkGoroutineExists function ?

Reply:
Yes.

```
[2021-12-12T12:11:56.974Z] executor/distsql_test.go:43:6: func `checkGoroutineExists` is unused (unused)

[2021-12-12T12:11:56.974Z] func checkGoroutineExists(keyword string) bool {

[2021-12-12T12:11:56.974Z]      ^

```

Reply:
![image](https://user-images.githubusercontent.com/44599853/145717514-3ec2ac85-0838-4d9a-a868-c09ab4062787.png)
I found checkGoroutineExists is called in line 123...

Reply:
I know. 

https://github.com/pingcap/tidb/blob/9cd0b49f7f5ca6d26e8e77e25ceda0afc970437b/executor/seqtest/seq_executor_serial_test.go#L750

there is the same function.

Reply:
how can I solve this , I tried to rename this function, it also prompted unused...

Reply:
Try to delete ```executor/distsql_test.go:43:6 checkGoroutineExists ``` ?

Reply:
make ```checkGoroutineExists``` public and delete the redundant.

Reply:
okay , let me try

Reply:
package name ```executor_test``` can be replaced by ```executor```.